front/src/applications/operationalStudies/views/SimulationResults.tsx

```diff
@@ -65,6 +66,13 @@ const SimulationResults = ({
     infraId
   );
 
+  const pathItemsCoordinates =
```

Comment:
Maybe a comment to explain why we do that

```suggestion
  // Compute path items coordinates in order to place them on the map
  const pathItemsCoordinates =
```

Reply:
done